Snakefile

```diff
@@ -193,11 +193,13 @@ rule process_airport_data:
         consumption_per_passenger="plots/results/consumption_per_passenger.png",
         correlation_matrix="plots/results/correlation_matrix.png",
         comparision_consumption_passengers="plots/results/comparision_consumption_passengers.png",
+        custom_airports_data=PYPSA_EARTH_DIR + "data/airports.csv",
     resources:
         mem_mb=3000,
     script:
         "plots/airport_data_postprocessing.py"
 
+ruleorder: process_airport_data > prepare_airports
```

Comment:
Will it work if we add if condition here? So it will prioritize `process_airport_data` if custom_airports are selected. Else prepare_airports are used.

Reply:
It should work. But I think a simpler option is to remove the custom airport config.